packages/core/src/App/Components/Layout/Footer/__tests__/toggle-fullscreen.spec.tsx

```diff
@@ -1,21 +1,26 @@
-// TODO refactor old tests in this component
 import React from 'react';
-import { ToggleFullScreen } from '../toggle-fullscreen.jsx';
-import { Icon } from '@deriv/components';
 import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ToggleFullScreen } from '../toggle-fullscreen.jsx';
 
 describe('ToggleFullScreen Component', () => {
-    it('should not have "ic-fullscreen--active" class when "is_full_screen" is false', () => {
+    it('should not have "ic-fullscreen--active" class when "is_full_screen" is "false"', () => {
         render(<ToggleFullScreen />);
-        const aElement = screen.getByTestId('dt_fullscreen_toggle');
-        expect(aElement).not.toHaveClass('ic-fullscreen--active');
+        const a_element = screen.getByTestId('dt_fullscreen_toggle');
+        expect(a_element).not.toHaveClass('ic-fullscreen--active');
     });
 
-    // it('should have "Full screen" text when "is_full_screen" is false', () => {
-    //     render(<ToggleFullScreen />);
-    // });
+    it('should have "Full screen" text when "is_full_screen" is "false"', () => {
+        render(<ToggleFullScreen />);
+        const popover_wrapper = screen.getByTestId('dt_popover_wrapper');
```

Comment:
There is no `dt_popover_wrapper` data-testid in the `toggle-fullscreen` component.

Reply:
It's in the popover component and we need that for reading the message